### PR TITLE
new course links for html, css and js

### PR DESCRIPTION
**Pull request by tanaytoshniwal**
added new course links for HTML, CSS, and Javascript.

### CODE REVIEW

**Review by John-pix — APPROVED** [review]
**This must be merged** because free code camps is a very nice platform. And all other links are videos, and some may not prefer videos at times because they are very large and difficult to keep up. So this article-based content may help

**Comment by tanaytoshniwal** [comment]
Can you please merge the PR, or do I need to do some changes?

File: programming-language-resources.md [hunk 1]
```diff
@@ -59,12 +59,15 @@
...
+    - [Basic HTML and HTML5](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/)
 - CSS
     - [Quick CSS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLr6-GrHUlVf8JIgLcu3sHigvQjTw_aC9C)
     - [CSS Crash Course for absolute Beginners](https://www.youtube.com/watch?v=yfoY53QXEnI)
+    - [Basic CSS](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/)
 - Javascript
     - [Learn Javascript in 12 minutes](https://www.youtube.com/watch?v=Ukg_U3CnJWI)
     - [Javascript Beginner Tutorials](https://www.youtube.com/playlist?list=PL41lfR-6DnOrwYi5d824q9-Y6z3JdSgQa)
+    - [Javascript Algorithms and Data Structures](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/)
 - Other Language 1
```
Comment (flux0806) [inline programming-language-resources.md:71]: [Python Full Course - Learn Python in 12 Hours | Python Tutorial For Beginners | Edureka](https://youtu.be/WGJJIrtnfpk)

Comment (John-pix) [reply programming-language-resources.md:71]: This is a pull request for adding links to html css and javascript course. But you have mentioned a python course. Maybe you can create a new branch and pull request for adding this link

**Comment by jwasham** [comment]
Thanks!

**Pull request merged**